test/commonTests.js

```diff
@@ -35,12 +34,12 @@ module.exports.createCommonTests = ({ Sentry, testkit }) => {
 
   test('should return the provided level', async function() {
     Sentry.configureScope(scope => {
-      scope.setLevel(Severity.Warning)
+      scope.setLevel('warning')
```

Comment:
why have we moved to hardcoded string?

Reply:
I assume because of this in the 7.0.0 release notes?: https://github.com/getsentry/sentry-javascript/releases/tag/7.0.0

>Deprecated Severity enum for bundle size reasons. [Please use string literals instead](https://github.com/getsentry/sentry-javascript/blob/7.0.0/MIGRATION.md#severity-severitylevel-and-severitylevels). (https://github.com/getsentry/sentry-javascript/pull/4926)